pkg/composableschemadsl/parser/parser_test.go

```diff
@@ -123,6 +123,10 @@ func TestParser(t *testing.T) {
 		{"arrow illegal function test", "arrowillegalfunc"},
 		{"caveat with keyword parameter test", "caveatwithkeywordparam"},
 		{"use expiration test", "useexpiration"},
+		// NOTE: a duplicate use statement should be an error at the level
+		// of the compiler, but should not be an error at the level
+		// of the parser.
+		{"duplicate use statement test", "duplicate_use_statement"},
```

Comment:
Adding a parsing test to verify that multiple use statements get parsed correctly. This was mostly a debugging tool in doing this work, but also describes desired behavior.